src/inspector/models/propertyitem.cpp

```diff
@@ -32,12 +32,10 @@ PropertyItem::PropertyItem(const mu::engraving::Pid propertyId, QObject* parent)
     m_propertyId = propertyId;
 }
 
-void PropertyItem::fillValues(const QVariant& currentValue, const QVariant& defaultValue)
+void PropertyItem::fillValues(const QVariant& currentValue)
 {
     updateCurrentValue(currentValue);
 
-    setDefaultValue(defaultValue);
-
     emit isModifiedChanged(isModified());
```

Comment:
Now that `isModified()` has its own member variable with its own setter, I think this signal emission is not necessary anymore, is it?

And that should make `fillValues` redundant, so that you can just use `updateCurrentValue`.